vendor/engines/projects/app/views/projects/projects/show.html.haml

```diff
@@ -9,8 +9,10 @@
 
   - @date_range_field = :ordered_at
   = render "shared/transactions/table",
-    order_details: @order_details
+    order_details: @order_details,
+    projects_table: true
```

Comment:
I think we could use `projects_table = controller_name == "ProjectsController"` instead of passing the local around - what do you think?